src/data.js

```diff
@@ -0,0 +1,10 @@
+export function fetchAllData(url) {
```

Comment:
mogoče bi lahko kar dedicated fn za fetchanje breedov naredu, ne splošno da passaš notr še url, ampak z hardcodanim urljem in je pol to `fetchBreeds` fn.. in v komponenti samo pokličeš `fetchBreeds`